issue_release_status.go

```diff
@@ -24,6 +24,12 @@ func any(vs []string, f func(string) bool) bool {
 	return false
 }
 
+func trimServiceName(original string) string {
```

Comment:
Maybe move this down to after is it used. Usually we try to structure code with most important code at the top and the by call order. So `Handle` uses `trimServiceName` so it should be defined in after but in proximity to `Handle`.

Reply:
Ah just a habit from C to place it before. Fix: https://github.com/lunarway/release-manager-bot/pull/23/commits/6d3686aa06053c8f95060180cb70db9c17c88113